addons/website_sale/static/src/js/website_sale_dashboard/website_sale_dashboard.dark.scss

```
@@ -0,0 +1,25 @@
+.o_dashboard_card {
+    &--purple {
+        background-color: $purple-200;
+
+        &.o_dashboard_card--active, &:hover, &:active {
+            background-color: $purple-300;
+        }
+    }
+
+    &--orange {
+        background-color: $orange-200;
+
+        &.o_dashboard_card--active, &:hover, &:active {
+            background-color: $orange-300;
+        }
+    }
+
+    &--cyan {
+        background-color: $cyan-200;
+
+        &.o_dashboard_card--active, &:hover, &:active {
+            background-color: $cyan-300;
+        }
+    }
+}
```

Only css variables in dark files :)
```suggestion
// = Website Sale Dashboard
// ============================================================================
// No CSS hacks, variables overrides only

.o_dashboard_card {
    --DashboardCard__purple-bg-color: #{$purple-200});
    --DashboardCard__purple-bg-color--active: #{$purple-300});
    ...
}
```